resources/CMakeLists.txt

```diff
@@ -111,6 +111,8 @@ set(ENERGYPLUS_OUTPUTS "")
 set(model_resources_src
   model/OpenStudio.idd
   model/schedulefile.csv
+
+  model/two_stories_pre_intersect.osm
```

Comment:
I just "touched" this file in `<root>/resources/model/two_stories_pre_intersect.osm` so obviously you need to replace with the actual file.

Reply:
I believe adding these osm files to this list will include them in the embedded file system. I'm not sure that is desired or necessary.

Reply:
This will just copy them to the `build/resources` directory, not embed them.